web-components/src/components/icons/icons.stories.tsx

```diff
@@ -126,6 +127,9 @@ export const linkedInIcon = (): JSX.Element => <LinkedInIcon />;
 
 export const linkIcon = (): JSX.Element => <LinkIcon color="black" />;
 
+/** this isn't really an icon, but it makes sense here I think in storybook */
+export const loadingIcon = (): JSX.Element => <Loading />;
+
```

Comment:
this is a bit weird to put that inside icons.stories.tsx while the component live somewhere else
why not just having it in icons/ folder?

Reply:
I moved this to a `loading.stories.tsx` inside of the `/loading` folder - given the talk about using skeleton components, I figure they could live under this folder as well